src/base/projects/Blob.svelte

```diff
@@ -1,49 +1,63 @@
 <script lang="ts">
   import type { Blob } from "@app/project";
+  import type { MaybeHighlighted } from "@app/syntax";
   import type { ProjectRoute } from "@app/router/definitions";
 
   import HeaderToggleLabel from "@app/base/projects/HeaderToggleLabel.svelte";
-  import ProjectLink from "@app/router/ProjectLink.svelte";
   import Readme from "@app/base/projects/Readme.svelte";
+  import { afterUpdate, beforeUpdate, onMount } from "svelte";
+  import { highlight } from "@app/syntax";
   import { isMarkdownPath, scrollIntoView, twemoji } from "@app/utils";
-  import { onMount } from "svelte";
+  import { lineNumbersGutter } from "@app/syntax";
+  import { toHtml } from "hast-util-to-html";
   import { updateProjectRoute } from "@app/router";
 
   export let activeRoute: ProjectRoute;
   export let blob: Blob;
   export let getImage: (path: string) => Promise<Blob>;
   export let line: string | undefined = undefined;
 
-  $: lineNumber = line ? parseInt(line.substring(1)) : undefined;
-
+  const fileExtension = blob.path.split(".").pop() ?? "";
   const lastCommit = blob.info.lastCommit;
-  const lines = blob.binary ? 0 : (blob.content.match(/\n/g) || []).length;
-  const hasFinalNewline = blob.content.endsWith("\n");
-  const lineNumbers = Array(hasFinalNewline ? lines : lines + 1)
-    .fill(0)
-    .map((_, index) => (index + 1).toString());
   const parentDir = blob.path
     .match(/^.*\/|/)
     ?.values()
     .next().value;
+  let content: MaybeHighlighted = undefined;
+
+  // Removing any existing highlights, before updating the component
```

Comment:
Could you maybe add more context on when this would happen?

Reply:
6026fbf4